### PR TITLE
remove duplicated method in pencilcodeembed.js

### PR DESCRIPTION
**Pull request by aug2uag**

### CODE REVIEW

**Comment by aug2uag** [comment]
Let me start out by saying that I hate to comment out tests, however, in this case their remedy was uncertain. My box can’t run the pencilcode environment with reasonable effort.

The goal for me is to have an independent distribution of the `pencilcodeembed.js` driven form of the magic `frame` the single page application relies upon. The suggestions then are biased towards that end.

The rationale for an independent distribution of the `pencilcodeembed.js` is to avoid dependency crashes in the future if goodness forbid something adverse should happen to any branch of the development and production of the Pencil environment.

Only after working with a clean image, and following Prequisite instructions re- assigning root to `/local/bin`, the modules installed correctly. The single culprit preventing a smooth installation on my native local environment was `fs-ext` or other third-party `fs` dependencies that may be raised as another issue elsewhere.

Nonetheless, even after achieving the most ideal environment reasonably available to me, the Grunt build and test environment were unmanageable to say the least.

Uglification of `editor.js` would cause Grunt to fail out. Inspection of this file reveals it is enormous; and why I removed the file from my own environment in order to proceed with testing; and I included this change with the current pull request.

My goal was to remedy the ‘it should be able to switch to single-pane mode’ in `edit_code.js` file of the test directory. The efforts to reach a starting point to access the test using the aforementioned clean image environment (i.e. Ubuntu 14.04.4 LTS (GNU/Linux 3.13.0-79-generic x86_64)) were not reasonable; and this raises the current issue to virtualize the PencilCode via Docker or other. Among others, I had the need to add the `page.open()` of Phantom as a decorator to `page.includeJs("http://ajax.googleapis.com/ajax/libs/jquery/1.6.1/jquery.min.js”)` in order for the Phantom socket not to crash. This modification was not tolerated, and resulted in identical time out errors generated from `test/utils/libutil.js`.

I’ve previously had success with VirtualBox and Vagrant environment locally. However, the issues mentioned here would continue to exist without explicit instructions to ensure normalization of environments.


Since I was not able to reliably run tests, the failing tests were temporarily commented out until other efforts are made to correct them or the normalized environment (i.e. with instructions). This was done over a series of commits, and utilizing Travis for feedback.

I noticed other tests were previously commented out, and not remedied (e.g. `test/browse_user.js:108`).

It’s therefore proposed that a configurable `editor.js` would exist from CDN for future works. Furthermore, that the `frame` magic would also be configurable. This would satisfy my goal for failsafe deployments of future applications relying on PencilCode.

Please let me know your thoughts regarding the suggestions above. I don’t know whether I’ve made some mistake that others handled themselves previously or avoided. It would go a long way towards maintaining ensurability of access to children and adults who rely on the services. Thanks in advance, and for all your efforts placed in the amazing PencilCode.
